examples/mysql-galera/pxc-node3-controller.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    name: pxc-node3 
+  name: pxc-node3 
+spec:
+  replicas: 1
+  selector:
+    name: pxc-node3 
```

Comment:
I'll fix this. I might have been looking at another example assuming I needed this.